src/config/constants.ts

```diff
@@ -15,10 +16,10 @@ export const BEAMER_ID = process.env.NEXT_PUBLIC_BEAMER_ID || ''
 
 // Wallets
 export const WC_BRIDGE = process.env.NEXT_PUBLIC_WC_BRIDGE || 'https://safe-walletconnect.gnosis.io/'
-export const FORTMATIC_KEY = process.env.NEXT_PUBLIC_FORTMATIC_KEY || ''
-export const PORTIS_KEY = process.env.NEXT_PUBLIC_PORTIS_KEY || ''
```

Comment:
We should check whether we're paying for these at some point as well. We should cancel them if so.

Reply:
We're definitely not paying for them.